models/intermediate/int_iterable__list_user_unnest.sql

```diff
@@ -71,9 +71,13 @@ with user_history as (
         is_current,
         email_list_ids,
         case when email_list_ids != '[]' then
-        {% if target.type == 'snowflake' %}
-        email_list_id.value
-        {% else %} email_list_id {% endif %} else null end as email_list_id
+            {% if target.type == 'snowflake' %}
+            email_list_id.value
+            {% elif target.type in ('spark','databricks') %}
+            email_list_id.col
```

Comment:
Can you explain why databricks and spark are using the `col` column? Where is this coming from?

Reply:
Yes. From [the docs](https://docs.databricks.com/sql/language-manual/functions/explode_outer.html#returns):
> The column produced by `explode_outer` of an array is named `col`. 

I was thinking that seemed reasonable based on how snowflake appears to do a similar thing on lines 74-75.

Reply:
Thanks for the explanation, that is what I assumed but just wanted to make sure. Looks good to me!